### ISSUE TITLE
Mapped path scripts not found

### ISSUE THREAD

**Issue by GitPuffy** [rbeer/nodebb-plugin-smoothshorts#8]
I am seeing this in my NodeBB logs.  smoothshorts version `0.2.1`, NodeBB `v1.0.2`

```
15/4 16:08 [10703] - warn: [plugins/nodebb-plugin-smoothshorts] Mapped path 'scripts => /home/centos/nodebb_installation/devNodeBB/node_modules/nodebb-plugin-smoothshorts/static/scripts/' not found.
```

I am not sure if this is an issue or not.

**Comment by rbeer** [comment]
Nah, no need to worry. :smile:

The warning just tells that there's a path defined in [plugin.json](https://github.com/rbeer/nodebb-plugin-smoothshorts/blob/master/plugin.json#L12) that's not existing. NodeBB ignores it then. Which would be a problem, normally. But the scripts that will go in there are still part of the [template file](https://github.com/rbeer/nodebb-plugin-smoothshorts/blob/master/public/templates/admin/plugins/smoothshorts.tpl#L91-L168). Seems I went ahead of myself and already made the necessary changes for the backend/ACP rework coming with `0.3.0`.

It's already changed in [0.3.0-pre](https://github.com/rbeer/nodebb-plugin-smoothshorts/tree/0.3.0-pre). Watch out with those -pre branches, though. I tend to get lazy and --force push into them, from time to time. :smirk:

**Comment by rbeer** [comment]
Think I should leave this open, until `master` is updated. :smiley_cat:

**Comment by rbeer** [comment]
published in 0.3.0 / 1386a2a
